src/content/app/tools/blast/components/blast-species-selector/BlastSpeciesSelector.scss

```diff
@@ -0,0 +1,118 @@
+@import 'src/styles/common';
+
+.header {
+  display: flex;
+  width: 860px;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 28px;
+  padding-bottom: 12px;
+}
+
+.headerGroup {
+  display: flex;
+  align-items: center;
+}
+
+.headerTitle {
+  font-size: 15px;
+  font-weight: $bold;
+}
+
+.maxSpecies {
+  font-weight: $light;
+  margin-right: 30px;
+}
+
+.clearAll {
+  margin-right: 30px;
+  color: $blue;
+  cursor: pointer;
+  user-select: none;
+}
+
+.speciesCounter {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 28px;
+  height: 28px;
+  color: $white;
+  background-color: $dark-grey;
+  font-weight: $bold;
+  border-radius: 100%;
+  font-size: 15px;
+  margin: 0 0.6rem 0 0.6rem;
+}
+
+.speciesSelectorContainer {
+  width: 860px;
+  height: 96%;
+  background-color: $light-grey;
+  border: 1px solid $grey;
+  padding: 10px;
+
+  .checkboxUnchecked {
+    margin-top: 0px;
+    margin-left: 10px;
+    width: 13px;
+    height: 13px;
+  }
+
+  .checkboxChecked {
+    margin-top: 0px;
+    margin-left: 10px;
+    width: 13px;
+    height: 13px;
+  }
+}
+
+.sequencesButton {
+  margin-left: 66px;
+}
+
+.speciesSelectorTable {
+  width: 100%;
+  text-align: left;
+  border-collapse: separate;
+  border-spacing: 0;
+  padding-left: 20px;
+  padding-right: 20px;
+
+  thead th {
+    border-bottom: 2px solid $blue;
+    padding: 6px 10px 6px 15px;
+    font-weight: 300;
+    font-size: 11px;
+  }
+
+  tbody td {
+    border-bottom: 1px solid $medium-light-grey;
+    padding: 14px 8px 14px 15px;
+  }
+}
+
+.commonNameHeader {
+  width: 240px;
+}
+
+.scientificNameHeader {
+  width: 260px;
+}
+
+.scientificNameCol {
+  font-style: italic;
+}
+
+.assemblyCol {
+  font-size: 11px;
+}
+
+.assemblyHeader {
+  width: 235px;
+}
+
+thead th.selectCol {
```

Comment:
That would be applied to all table heads 😕  Surely you only want the thead of the `.speciesSelectorTable`

Reply:
No it wont apply to all table heads but to all table heads with a class "selectCol" to th. But i have moved it under the `.speciesSelectorTable` class to make it clearer.